README.md

```diff
@@ -144,16 +144,17 @@ _Note: Links are provided further down to create an app for your logged in user
 * Metadata (read)
 
 **Organization Permissions**
+
 * Self-hosted runners (read / write)
 
 _Note: All API routes mapped to their permissions can be found [here](https://docs.github.com/en/rest/reference/permissions-required-for-github-apps) if you wish to review_
 
 **Subscribe to events**
 
-At this point you have a choice of configuring a webhook, a webhook is needed if you are going to use [webhook driven scaling](#webhook-driven-scaling). The webhook can be configured centrally in the GitHub app itself or separately. In either case the event details are:
+At this point you have a choice of configuring a webhook, a webhook is needed if you are going to use [webhook driven scaling](#webhook-driven-scaling). The webhook can be configured centrally in the GitHub app itself or separately. In either case you need to subscribe to the same events:
 
-* Check run (required for all webhook driven scaling events)
-* Workflow job (optionally) (required for [webhook driven scaling with workflow_job events](https://github.com/actions-runner-controller/actions-runner-controller#example-1-scale-on-each-workflow_job-event)
+* Check run
+* Workflow job
```

Comment:
```suggestion
* Workflow job
* Check run
```

I think we'd generally recommend `Workflow job` first so that users won't try `check run` and experience various gotchas of it.

Or, perhaps we'd better remove the reference to `check run` at all, assuming all the GHES in the world has already migrated to versions that supports `workflow_job` events? 😁